stable/wordpress/values.yaml

```diff
@@ -106,21 +106,41 @@ healthcheckHttps: false
 ## ref: http://kubernetes.io/docs/user-guide/ingress/
 ##
 ingress:
-    enabled: false
-    hostname: wordpress.local
+  ## Set to true to enable ingress record generation
+  enabled: false
 
-    ## Ingress annotations
-    ##
-    # annotations:
-    #   kubernetes.io/ingress.class: nginx
+  ## The list of hostnames to be covered with this ingress record.
+  ## Most likely this will be just one host, but in the event more hosts are needed, this is an array
+  hosts:
+  - name: wordpress.local
+
+    ## Set this to true in order to enable TLS on the ingress record
+    ## A side effect of this will be that the backend wordpress service will be connected at port 443
+    tls: false
 
-    ## Ingress TLS configuration
-    ## Secrets must be manually created in the namespace
+    ## If TLS is set to true, you must declare what secret will store the key/certificate for TLS
+    tlsSecret: wordpress.local-tls
+
+    ## Ingress annotations done as key:value pairs
+    ## If you're using kube-lego, you will want to add:
+    ## kubernetes.io/tls-acme: true
+    ##
+    ## For a full list of possible ingress annotations, please see
+    ## ref: https://github.com/kubernetes/ingress-nginx/blob/master/docs/annotations.md
     ##
-    # tls:
-    #   - secretName: wordpress.local-tls
-    #     hosts:
-    #       - wordpress.local
+    ## If tls is set to true, annotation ingress.kubernetes.io/secure-backends: "true" will automatically be set
+    annotations:
```

Comment:
I don't think you meant for `annotations` and `secrets` to be part of the `hosts` array

Reply:
no, its intentional

Reply:
:+1: I understand the need for multiple Ingress objects based on the other response.